BUILD.md

```diff
@@ -376,11 +376,12 @@ stored inside the build directory, as either of:
 | Option | Default Value | Description |
 | --- | ---| ---|
 | `assert` | OFF | Enable assertions.
-| `reporting` | OFF | Build the reporting mode feature. |
 | `coverage` | OFF | Prepare the coverage report. |
-| `tests` | ON | Build tests. |
-| `unity` | ON | Configure a unity build. |
+| `reporting` | OFF | Build the reporting mode feature. |
```

Comment:
I assume this is later removed in the PR that removes reporting mode?

Reply:
Yes, though it will need to be merged with this change after it hits `develop`.